AlgorithmFactory/Loader.cs

```diff
@@ -315,7 +309,12 @@ public static List<string> GetExtendedTypeNames(Assembly assembly)
                 }
                 catch (ReflectionTypeLoadException e)
                 {
-                    assemblyTypes = e.Types;
+                    assemblyTypes = e.Types.Where(t => t != null).ToArray();
+
+                    foreach (Exception inner in e.LoaderExceptions)
+                    {
+                        Log.Error($"Loader.GetExtendedTypeNames(): {inner.Message}");
```

Comment:
Do we also want to log `e.Message` here as well?

Reply:
This message always says: **Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.'**  I will include similiar message to the logs along with count of the null entries .